### DISCUSSION TITLE
Pre-release 0.3.0

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#115] [Announcements]
## If you have been experiencing problems with pre-0.2.x Whisky, upgrade to this version

**Quality of Life:**
- Added Danish
- Logging improvements

**New Features:**
- Bottles can now be created in any directory
- External bottles can now be imported
- Wine and GPTK are installed in Application Support

**Bug Fixes:**
- Fixes Steam
- Fixes MSI selection
- Fixes an issue where Whisky would invalidate its signature, preventing it from accessing files

**Full Changelog**: https://github.com/IsaacMarovitz/Whisky/compare/pre-0.2.4...pre-0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/IsaacMarovitz/Whisky/releases/tag/pre-0.3.0'>Pre-release 0.3.0</a>.</em>

**Comment by EatingIting** [comment]
I was click the run but, error occured.

error message is Failed to open program!

Failed to open SteamSetup.exe: The operation couldn’t be completed. (Swift.String error 1.)

How can i run Steam and Battle-net?
